solarwinds_apm/configurator.py

```diff
@@ -87,6 +95,10 @@ def _configure(self, **kwargs: int) -> None:
         apm_txname_manager = SolarWindsTxnNameManager()
         apm_fwkv_manager = SolarWindsFrameworkKvManager()
         apm_config = SolarWindsApmConfig()
+        # TODO add args
+        # https://swicloud.atlassian.net/browse/NH-68264
+        # See also SolarWindsApmConfig._get_extension_components
```

Comment:
If not agent_enabled or not is_lambda, then this is no-op. Defined [here](https://github.com/solarwinds/apm-python/blob/66a6e60d9d6911a05921b22eb00bfdbfe8173ed3/solarwinds_apm/apm_config.py#L156-L166):

```
agent_enabled T, is_lambda F -> c-lib extension, c-lib Context, no-op settings API
agent_enabled T, is_lambda T -> no-op extension, no-op Context, c-lib settings API
agent_enabled F              -> all no-op
```